tests/installation/partitioning_lvm.pm

```diff
@@ -51,11 +51,23 @@ sub run {
         }
     }
 
-    send_key "alt-d";
+    # Storage NG introduces a new partitioning dialog. partitioning.pm detects this by the existence of the "Guided Setup" button
+    # and sets the STORAGE_NG variable. This button uses a new hotkey.
+    if (get_var("STORAGE_NG")) {
+        send_key "alt-g";
+    }
+    else {
+        send_key "alt-d";
+    }
 
-    if (check_screen('inst-partition-radio-buttons', 10)) {    # detect whether new (Radio Buttons) YaST behaviour
+    my $numdisks = get_var("NUMDISKS");
+    print "NUMDISKS = $numdisks\n";
+
+    check_screen([qw(inst-partition-radio-buttons inst-partition-guided inst-partitioning-scheme)], 10);
```

Comment:
Please don't use check_screen with timeout other than 0. Either surround the previous `send_key` with `wait_screen_change` or use a multi-tag assert_screen plus `match_has_tag`